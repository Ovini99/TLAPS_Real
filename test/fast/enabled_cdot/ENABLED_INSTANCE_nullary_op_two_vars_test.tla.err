@!!BEGIN
@!!type:obligationsnumber
@!!count:0
@!!END

File "/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../library/TLAPS.tla", line 1, character 1 to line 363, character 77:
[INFO]: All 0 obligation proved.
@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:16:1:16:3
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligation
@!!id:2
@!!loc:16:4:16:17
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligationsnumber
@!!count:2
@!!END

** Unexpanded symbols: ---

@!!BEGIN
@!!type:obligation
@!!id:2
@!!loc:16:4:16:17
@!!status:trivial
@!!prover:tlapm
@!!already:false
@!!END

Executable "isabelle-process" not found in this PATH:
/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/lib/tlaps/bin
@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:16:1:16:3
@!!status:failed
@!!prover:smt
@!!meth:time-limit: 25; time-used: 0.0 (0%)
@!!reason:false
@!!already:false
@!!obl:
(* SMT failed with status = sat
 *)
ASSUME NEW VARIABLE x,
       TRUE 
PROVE  \E r#0, r#1 : r#1 # r#0

@!!END

@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:16:1:16:3
@!!status:failed
@!!prover:zenon
@!!meth:time-limit: 50; time-used: 0.0 (0%)
@!!reason:false
@!!already:false
@!!obl:
ASSUME NEW VARIABLE x,
       TRUE 
PROVE  \E r#0, r#1 : r#1 # r#0

@!!END

@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:16:1:16:3
@!!status:failed
@!!prover:isabelle
@!!meth:auto; time-limit: 150
@!!reason:Executable "isabelle-process" not found in this PATH:
/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/lib/tlaps/bin

@!!already:false
@!!obl:
ASSUME NEW VARIABLE x,
       TRUE 
PROVE  \E r#0, r#1 : r#1 # r#0

@!!END

File "./ENABLED_INSTANCE_nullary_op_two_vars_test.tla", line 16, characters 1-2:
[ERROR]: Could not prove or check:
           ASSUME NEW VARIABLE x,
                  M!A == ?EnabledWrapper(LAMBDA r#0, r#1 : r#1 # r#0),
                  ExpandENABLED 
           PROVE  M!A
File "./ENABLED_INSTANCE_nullary_op_two_vars_test.tla", line 1, character 1 to line 18, character 58:
[ERROR]: 1/2 obligations failed.
There were backend errors processing module `"ENABLED_INSTANCE_nullary_op_two_vars_test"`.
@!!BEGIN
@!!type:obligationsnumber
@!!count:0
@!!END

File "./ENABLED_INSTANCE_nullary_op_two_vars_test.tla", line 4, character 1 to line 9, character 22:
[INFO]: All 0 obligation proved.
