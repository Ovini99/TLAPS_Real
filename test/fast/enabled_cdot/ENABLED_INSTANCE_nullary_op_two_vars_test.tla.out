
\* TLAPM version 1.5.0
\* launched at 2023-11-27 14:14:21 with command line:
\* /home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../tlapm -I /home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../library --toolbox 0 0 --nofp ENABLED_INSTANCE_nullary_op_two_vars_test.tla --stretch 5

(* created new ".tlacache/TLAPS.tlaps/TLAPS.thy" *)
(* fingerprints written in ".tlacache/TLAPS.tlaps/fingerprints" *)
(* will not use fingerprints (because of option `--nofp`), but will now load fingerprints from the file `.tlacache/ENABLED_INSTANCE_nullary_op_two_vars_test.tlaps/fingerprints`, in order to overwrite with the new fingerprints, and then save the results at the end. *)
COMMAND-LINE ARGUMENT
file=.tlacache/ENABLED_INSTANCE_nullary_op_two_vars_test.tlaps/tlapm_e11fa9.smt; PATH='/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/lib/tlaps/bin'; z3 -smt2 -v:0 AUTO_CONFIG=false smt.MBQI=true "$file" >.tlacache/ENABLED_INSTANCE_nullary_op_two_vars_test.tlaps/tlapm_e11fa9.smt.out
CONTENT OF FILE SENT TO BACKEND
;; TLA+ Proof Manager 1.5.0
;; Proof obligation #1
;;   generated from file "./ENABLED_INSTANCE_nullary_op_two_vars_test.tla", line 16, characters 1-2

(set-logic UFNIA)
(declare-sort u 0)
;; Standard TLA+ operators

;; Terms, predicates and strings


;; Theorem to be Proved
(assert (not
  (exists ((a_CONSTANTunde_rhash_0unde_a u) (a_CONSTANTunde_rhash_1unde_a u))
    (not (= a_CONSTANTunde_rhash_1unde_a a_CONSTANTunde_rhash_0unde_a)))))

(check-sat)
(exit)

RESULT
sat

Zenon error: exhausted search space without finding a proof
(* created new ".tlacache/ENABLED_INSTANCE_nullary_op_two_vars_test.tlaps/ENABLED_INSTANCE_nullary_op_two_vars_test.thy" *)
(* fingerprints written in ".tlacache/ENABLED_INSTANCE_nullary_op_two_vars_test.tlaps/fingerprints" *)
(* created new ".tlacache/Inner.tlaps/Inner.thy" *)
(* fingerprints written in ".tlacache/Inner.tlaps/fingerprints" *)
